frontend/src/modules/editor/contentObject/views/editorMenuView.js

```diff
@@ -114,28 +126,28 @@ define(function(require){
     */
     getItemHeirarchy: function(model, done) {
       var hierarchy = [];
-      if(model.get('_type') === 'menu') {
+      if (model.get('_type') === 'menu') {
         hierarchy.push(model);
       }
       var __this = this;
       var _getParent = function(model, callback) {
         var parent = __this.contentobjects.findWhere({ _id: model.get('_parentId') });
-        if(parent) {
+        if (parent) {
           hierarchy.push(parent);
           return _getParent(parent, callback);
         }
         hierarchy.push(Origin.editor.data.course);
         callback();
       };
       _getParent(model, function() {
-        if(typeof done === 'function') done.call(__this, hierarchy.reverse());
+        if (typeof done === 'function') done.call(__this, hierarchy.reverse());
       });
     },
 
     onSelectedItemChanged: function(model) {
-      if(model.get('_id') === Origin.editor.currentContentObject && Origin.editor.currentContentObject.get('_id')) {
-        return;
-      }
+      if (!model || !Origin.editor.currentContentObject) return;
```

Comment:
Apologies, this is causing the code to block items being selected for the first time something's clicked (as Origin.editor.currentContentObject will always be undefined)